components/camel-rabbitmq/src/main/java/org/apache/camel/component/rabbitmq/RabbitMQEndpoint.java

```diff
@@ -919,4 +943,9 @@ public void setExclusive(boolean exclusive) {
         this.exclusive = exclusive;
     }
 
+    public boolean isLenientProperties() {
+        // true to allow dynamic URI options to be configured
+        return true;
```

Comment:
I dont think it ought to be lenient when all the new options are prefixed with arg.